src/lib/mdns/Resolver.h

```diff
@@ -100,6 +102,66 @@ struct DiscoveredNodeData
     DiscoveredNodeData() { Reset(); }
     bool IsHost(const char * host) const { return strcmp(host, hostName) == 0; }
     bool IsValid() const { return !IsHost("") && ipAddress[0] != chip::Inet::IPAddress::Any; }
+    void Print() const
+    {
+        if (rotatingIdLen > 0)
+        {
+            char rotatingIdString[chip::Mdns::kMaxRotatingIdLen * 2 + 1] = "";
+            Encoding::BytesToUppercaseHexString(rotatingId, rotatingIdLen, rotatingIdString, sizeof(rotatingIdString));
+            printf("Rotating ID: %s\n", rotatingIdString);
```

Comment:
should these be ChipLogInfo calls rather than printf?

Reply:
Switched to ChipLogProgress.